pyre/extra_tests/snippets/module_type.py

```
@@ -39,6 +39,21 @@
 assert fresh.value == 42
 assert vars(fresh)["value"] == 42
 
+
+class InitializingFlag:
+    def __bool__(self):
+        raise RuntimeError("initializing truth test")
+
+
+fresh.__spec__ = types.SimpleNamespace(_initializing=InitializingFlag())
+fresh.__annotate__ = lambda format: {"value": int}
```

Ruff (0.16.2)

[error] Lambda argument `format` is shadowing a Python builtin

(A006)